hack/bin/entrypoint.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
```

Comment:
Did you mean to commit these files? They look like they have nothing to do with wire-docs

Reply:
No, it seems I accidentaly staged them. Thanks for checking

Reply:
I recreated the PR from scratch